package.json

```diff
@@ -24,35 +24,35 @@
     "url": "https://github.com/nextcloud/nextcloud-moment"
   },
   "dependencies": {
-    "@nextcloud/l10n": "1.4.1",
-    "core-js": "3.18.2",
+    "@nextcloud/l10n": "^1.4.1",
+    "core-js": "^3.21.1",
     "jed": "^1.1.1",
-    "moment": "2.29.1",
+    "moment": "^2.29.2",
```

Comment:
**Suggestion**: put moment as [Peer Dependencies](https://nodejs.org/uk/blog/npm/peer-dependencies/) and enable projects who use `@nextcloud/moment` package to handle future security issues without waiting for a patch from this package.

Reply:
I thought about that, too. But as far as I know, this requires npm 7 and we do not have this requirement in `@nextcloud/moment`, now. Therefore, I decided to make this PR backwards compatible in order to get it released as soon as possible.

Reply:
From NPM post @korelstar 


> npm versions 1, 2, and 7 will automatically install peerDependencies if they are not explicitly depended upon higher in the dependency tree. For npm versions 3 through 6, you will receive a warning that the peerDependency is not installed instead.


Of course, this will be a small breaking change but looks reasonable for me.